src/Microsoft.ML.Transforms/Text/WordBagTransform.cs

```diff
@@ -220,7 +220,7 @@ internal bool TryUnparse(StringBuilder sb)
 
         /// <summary>
         /// This class is a merger of <see cref="ValueToKeyMappingTransformer.Options"/> and
-        /// <see cref="NgramExtractingTransformer.Arguments"/>, with the allLength option removed.
+        /// <see cref="NgramExtractingTransformer.Options"/>, with the allLength option removed.
         /// </summary>
         public abstract class ArgumentsBase
```

Comment:
>public [](start = 8, length = 6)

internal?

Reply:
fixed

---
In reply to: [254030994](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254030994) [](ancestors = 254030994)